tests/recipes/cad1/task1/test_enhance.py

```diff
@@ -0,0 +1,162 @@
+"""Tests for the enhance module"""
+# pylint: disable=import-error
+
```

Comment:
Can make it just a fraction easier to work with resource files if we define a `RESOURCES` object of type `Path()`.
```suggestion
from pathlib import Path
```